src/components/card/card.module.scss

```diff
@@ -12,6 +12,7 @@
   padding-top: 56.6%;
   display: block;
   margin-bottom: 1.5rem;
+  width: 110%;
```

Comment:
Is this correct? Why do you need more than 100%?

Reply:
I was working a lot in this, I need the picture bigger than 100, but, If I make the container <article> bigger, the picture doesn't look like if I apply 110 to the picture. If you don't understand, I can try to explain this differently.